src/Linux/Native/RunAsUser/src/main.cpp

```diff
@@ -49,8 +50,10 @@ static constexpr char RTVS_RESPONSE_TYPE_RTVS_ERROR[] = "rtvs-error";
 
 static constexpr char RTVS_MSG_AUTH_ONLY[] = "AuthOnly";
 static constexpr char RTVS_MSG_AUTH_AND_RUN[] = "AuthAndRun";
+static constexpr char RTVS_MSG_KILL_PROCESS[] = "KillProcess";
 
 static constexpr char RTVS_RHOST_PATH[] = "/usr/lib/rtvs/Microsoft.R.Host";
+static constexpr char RTVS_KILL_PATH[] = "/bin/kill";
```

Comment:
It feels like it would be better as a more generic API, so that the broker can use it more flexibly as and when it needs to. What's the downside? If it's mainly security, then, given that the host can run arbitrary code (including spawning arbitrary child processes), I think it's moot anyway.